h5py/h5t.pyx

```diff
@@ -1006,6 +1006,8 @@ cdef class TypeCompositeID(TypeID):
         """
         return H5Tget_member_index(self.id, name)
 
+def get_third(iterable):
```

Comment:
I figured it didn't save or help anything and perhaps even made it slower, so I didn't mention this previously.  Not like I benchmarked it tho.